### ISSUE TITLE
[Feature] groupfree3d objectness loss

### ISSUE THREAD

**Issue by yehx1** [open-mmlab/mmdetection3d#2189]
### What's the feature?

groupfree3d_head.py line 441 and line 445
when computing objectness loss, it uses 1 - obj_targets. Is it better than just using obj_targets?

### Any other context?

sampling_objectness_loss = self.sampling_objectness_loss(
            sampling_obj_score,
            1 - sampling_targets.reshape(-1),
            sampling_weights.reshape(-1),
            avg_factor=batch_size)

objectness_loss = self.objectness_loss(
                obj_score.reshape(-1, 1),
                1 - objectness_targets.reshape(-1),
                objectness_weights.reshape(-1),
                avg_factor=batch_size)

**Comment by JingweiZhang12** [comment]
Maybe it's the original implementation of the official code and we didn't modify it.

**Comment by zyrant** [comment]
the original implementation of the official code is in https://github.com/zeliu98/Group-Free-3D/blob/ef8b7bb5c3bf5b49b957624595dc6a642b6d0036/models/loss_helper.py#L59, and it seems that it didn't use `1 - obj_targets`.

**Comment by Agito555** [comment]
It's really quite confusing. If I understand correctly, in 'sampling_targets', 1 represents whether it is one of the nearest objects to the ground truth (GT), and 0 represents that it's not. On the other hand, 'obj_score' is the logits for whether it's one of the nearest objects to the GT. I don't understand why we're using '1 - sampling_targets' for supervision.
